Makefile

```diff
@@ -69,11 +69,19 @@ else
 endif
 
 test: clean setup check_no_test_deps $(BIN)/thrift
+	$(MAKE) test_vanilla
+	$(MAKE) test_relay_frame_leaks
+
+test_vanilla:
 	@echo Testing packages:
```

Comment:
maybe call out this as `Testing packages without relay frame leaks! To enable, use test_relay_frame_leaks`.

Reply:
i considered that actually, but that would also make the echo show up in `make test`, i.e. something like
```
$ make test
Testing packages without relay frame leaks! To enable, use test_relay_frame_leaks
Running frame pool tests
Testing relay frame leaks
```
which would be pretty confusing. wdyt?

Reply:
rough call hehe. If you read the manual before you run, comment is better. If you don't bother to read and just run, printing out as echo is better. I'm fine with the way it is.